DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -420,6 +420,11 @@ protected function getMappingResourceExtension()
         return 'mongodb';
     }
 
+    protected function getMetadataDriverClass(string $driverType) : string
+    {
+        return '%' . $this->getObjectManagerElementName('metadata.' . $driverType . '.class%');
```

Comment:
Little nitpick: I would prefer the `%` at the end outside of the method `getObjectManagerElementName()`. The call depends on the implementation that the argument will be appended on whatever the method does inside.

```suggestion
        return '%' . $this->getObjectManagerElementName('metadata.' . $driverType . '.class') . '%';
```